doc/developer_guide/developer_guide.md

```diff
@@ -21,7 +25,7 @@ bash install.sh
 
 ## Design Goals:
 
-We use AWS as backend, because it provides the possibility to run the whole workflow during a ci-test.
+script-languages-developer-sandbox use AWS as backend, because it provides the possibility to run the whole workflow during a ci-test.
```

Comment:
```suggestion
script-languages-developer-sandbox uses AWS as backend, because it provides the possibility to run the whole workflow during a ci-test.
```